features/step_definitions/barclamp_rabbitmq/rabbitmq_steps.rb

```diff
@@ -0,0 +1,74 @@
+Given(/^the barclamp proposal is using rabbitmq as messaging backend$/) do
+    json_response = JSON.parse(admin_node.exec!("crowbar rabbitmq show default").output)
+    expect(json_response["attributes"]["rabbitmq"]).not_to be_empty
+end
+
+When(/^the node with rabbitmq-server role has been detected successfully$/) do
+    json_response = JSON.parse(admin_node.exec!("crowbar rabbitmq show default").output)
+    @node_fqdn = json_response["deployment"]["rabbitmq"]["elements"]["rabbitmq-server"].first
+    expect(@node_fqdn).not_to be_empty
+end
+
+Then(/^I can check the status of rabbitmq-server using command line$/) do
+    rabbitmq_status = admin_node.exec!("rabbitmqctl status").output.lines[1..-1] 
```

Comment:
Actually, admin node is acting as a client while the control node hosts the rabbitmq server. Here i am trying to check if the server hosted is accessible from an external client admin node was convenient.